websubhub-examples/kafka-hub/hub/Config.toml

```diff
@@ -0,0 +1,30 @@
+[kafkaHub.config]
+# Flag to check whether to enable/disable security
+SECURITY_ON = true
+
+# Server Id which is to be used in a server cluster
```

Comment:
Isn't something like this better ?

"Server ID is is used to uniquely identify each server. Each server must have a unique name."